src/main/java/net/sf/jabref/logic/undo/UndoRedoEvent.java

```diff
@@ -0,0 +1,10 @@
+package net.sf.jabref.logic.undo;
+
+
+public class UndoRedoEvent extends AddUndoEvent {
```

Comment:
My head is spinning by thinking about UndoRedoEvents, as undo and redo should cancel each other out, leaving only a event? Can you please document this? And the AddUndoEvent als well? What should be described in these events? When are they fired?